### PR TITLE
small tweaks to aid tilt-based local development

### PR DESCRIPTION
**Pull request by ewollesen**

### CODE REVIEW

**Review by toddkazakov — APPROVED** [review]

File: charts/keycloak/dev-realm.json [hunk 1]
```diff
@@ -1515,13 +1515,10 @@
     "strictTransportSecurity" : "max-age=31536000; includeSubDomains"
   },
   "smtpServer" : {
-    "password" : "123456",
-    "auth" : "true",
-    "host" : "email-smtp.us-west-2.amazonaws.com",
+    "host" : "smtp",
```
Comment (darinkrauss) [inline charts/keycloak/dev-realm.json:1518]: Note that this file is a generated from Keycloak. This is an export from the Keycloak dev realm for local development. Any Keycloak changes (new versions, changes to default setup, extensions, etc.) will require that this file be re-exported and we'll lose these changes.

Comment (ewollesen) [reply charts/keycloak/dev-realm.json:1518]: Wow, I had no idea. Thanks for letting me know.

Comment (ewollesen) [reply charts/keycloak/dev-realm.json:1518]: With https://github.com/tidepool-org/ops/pull/300 and https://github.com/tidepool-org/terraform-modules/pull/73 this should no longer be a concern. A regenerated realm file should include this new config now.

**Review by darinkrauss — APPROVED** [review]
LGTM!

**Pull request merged**